add mit r&d ai post (#60)
DaanVanVugt/ignitioncomputing: _posts/news/2024-07-18-MIT R&D AI.md

---
title: MIT R&amp;D AI grant won with highest score in NL!
layout: featured
author: Yonis le Grand
image: /assets/img/news/GTM-X.gif
categories: news
---

We are proud to announce that our AI innovation project proposal “Quality Predictions and simulation optimisation for glass production” with [CelSian Glass & Solar BV](https://www.celsian.nl/) ranked highest of 57 applications for the MIT AI R&D program by the Netherlands Enterprise Agency (RVO).

> The SME Innovation Stimulus for Regional and Top Sectors (MIT in Dutch) encourages SMEs to develop creative initiatives. These top ten sectors in the Netherlands have been designated as those that have the potential to solve global problems and help boost the country’s economy and competitiveness. [The MIT R&D AI is the dedicated stimulus program for innovative collaboration projects by SMEs to deploy innovative AI tools in the top sectors](http://ained.nl/ained-geeft-stevige-impuls-aan-mkb-voor-ontwikkeling-en-toepassing-van-ai-2/ ).

Reviewer: ```suggestion
The SME Innovation Stimulus for Regional and Top Sectors (MIT in Dutch) encourages Small and Medium-sized Enterprises (SMEs) to develop creative initiatives. These top ten sectors in the Netherlands have been designated as those that have the potential to solve global problems and help boost the country’s economy and competitiveness. [The MIT R&D AI is the dedicated stimulus program for innovative collaboration projects by SMEs to deploy innovative AI tools in the top sectors](http://ained.nl/ained-geeft-stevige-impuls-aan-mkb-voor-ontwikkeling-en-toepassing-van-ai-2/ ).
```

Our project proposal was graded on objective criteria on its potential, innovativeness, sustainability and feasibility. It received excellent marks (94.9/100 points) from the RVO. This grant will allow us to develop simulation acceleration tools for complex multiphysics problems, and deploy them in the glass industry.

> Our technology- partner CelSian has decades of experience in CFD tools used for the quality prediction of glass making. We will be integrating and adapting Preconnet with CelSians market-leading GTM-X tool (see article picture). Preconnet is a machine-learning based tool to accelerate simulations by generating preconditioners, initial guesses and proposing solver configurations. Preconnet has been developed for plasma simulations in a previous project with [Plasimo](/posts/news/2022/01/10/Preconnet_Teamup) and is a versatile tool.

Reviewer: ```suggestion
Our technology- partner CelSian has decades of experience in CFD tools used for the quality prediction of glass making. We will be integrating and adapting Preconnet with CelSian's market-leading GTM-X tool (see article picture). Preconnet is a machine-learning based tool to accelerate simulations by generating preconditioners, initial guesses and proposing solver configurations. Preconnet is a versatile tool that was developed for plasma simulations in a previous project in collaboration with [Plasimo](/posts/news/2022/01/10/Preconnet_Teamup). Its functionality, however, can be extended to other fields as well.
```

Reviewer 2: Als je zegt dat preconnet versatile is, dan hoef je toch niet te vertellen dat het in andere velden toepasbaar is?

> The glass-melting process can take up to multiple days. This makes glass production difficult. Temperature and material fluctuations have implications for the final product. Product quality issues, such as bubbles in glass, may arise. This can lead to the loss of large batches of glass, totaling multiple days of production. New developments in machine learning and simulation provide more accurate predictions of glass quality.
> The project is expected to result in improved quality prediction models leading to substantial economic and environmental gains. Through more efficient feed-stock use and energy consumption, each glass oven could save energy costs in the range of  € 250.000 to € 1.250.000 per year as well as reducing CO2 and nitrous oxide emissions. This undeniably positive expected impact fits with our vision of supporting the sustainable industry of the future.

Reviewer 2: ```suggestion
The glass-melting process can take up to multiple days. This makes glass production difficult. Temperature and material fluctuations have implications for the final product. Product quality issues, such as bubbles in glass, may arise. This can lead to the loss of large batches of glass, totaling multiple days of production. New developments in machine learning and simulation provide more accurate predictions of glass quality.

The project is expected to result in improved quality prediction models leading to substantial economic and environmental gains. Through more efficient feed-stock use and energy consumption, each glass oven could save energy costs in the range of  € 250.000 to € 1.250.000 per year as well as reducing CO2 and nitrous oxide emissions. This undeniably positive expected impact fits with our vision of supporting the sustainable industry of the future.
```
